tools/zip_examples.py

```diff
@@ -0,0 +1,86 @@
+# !python
+
+import argparse
+import logging
+import os
+import pprint
+import shutil
+import sys
+import tempfile
+import time
+import zipfile
+
+pp = pprint.PrettyPrinter(indent=4, width=100)
+
+
+def configure_logging(lvl=logging.WARNING, logfile=None):
+    root = logging.getLogger()
+    root.setLevel(lvl)
+
+    formatter = logging.Formatter("[%(asctime)s] [%(levelname)8s] --- %(message)s (%(filename)s:%(funcName)s:%(lineno)s)", "%Y-%m-%d %H:%M:%S")
+    if logfile is None:
+        hndlr = logging.StreamHandler(sys.stdout)
```

Comment:
`handler` seems better

Reply:
See #1238

Reply:
Can you use newly merged shared implementation?